packages/kbn-pm/src/production/integration_tests/build_production_projects.test.js

```diff
@@ -0,0 +1,44 @@
+import tempy from 'tempy';
+import copy from 'cpy';
+import { resolve } from 'path';
+import globby from 'globby';
+
+import { buildProductionProjects } from '../build_production_projects';
+import { getProjects } from '../../utils/projects';
+
+describe('kbn-pm production', function() {
```

Comment:
As we've moved this to jest, can you remove the `describe` wrapper and rename `it` to `test`?

Reply:
:heavy_check_mark: